src/python/pants/binaries/binary_util.py

```diff
@@ -209,3 +216,27 @@ def _fetch_binary(self, name, binary_path):
     logger.debug('Selected {binary} binary bootstrapped to: {path}'
                  .format(binary=name, path=bootstrapped_binary_path))
     return bootstrapped_binary_path
+
+  def _compare_file_checksums(self, filepath, checksum=None, digest=None):
```

Comment:
These edits should no longer be necessary because this will be located under the Task directory.